### PR TITLE
can now start / stop (as daemon)

### PR DESCRIPTION
**Pull request by klausfmh**
starting / stopping as daemon working now.

I used very simple but not very popular external module.

We have to decide whether we want to add it as dependency or just copy paste its code into the repo

### CODE REVIEW

File: pypeman/commands.py [hunk 1]
```diff
@@ -108,15 +114,66 @@ def main(debug_asyncio=False, profile=False, cli=False):
...
+            print("nodae")
+            main_func()
+
 
+@begin.subcommand
+def stop():
+    """ stops an already running pypeman instance """
+    def dummy():
+        pass
+    daemon = mk_daemon(dummy)
```
Comment (jrmi) [inline pypeman/commands.py]: use lambda function here.

File: pypeman/commands.py [hunk 2]
```diff
@@ -108,15 +114,66 @@ def main(debug_asyncio=False, profile=False, cli=False):
...
+
+    if reload:
+        print("RELOAD")
+        start_func()
+    else:
+        if daemon:
+            daemon = mk_daemon(main_func)
+            daemon.start()
+        else:
+            print("nodae")
```
Comment (jrmi) [inline pypeman/commands.py]: Remove print ? or make them homogeneous in Case.

File: pypeman/commands.py [hunk 3]
```diff
@@ -27,6 +27,12 @@
 import warnings
 from functools import partial
 
+# TODO: can remove try except if we have DaemonLite as a hard requirement
+try:
+    from DaemonLite import DaemonLite
```
Comment (jrmi) [inline pypeman/commands.py]: We can make it a hard requirements for now so you can remove this try/except.

File: requirements.txt [hunk 4]
```diff
@@ -1,2 +1,5 @@
 # Mandatory
 begins
+# rmv next line if we decie do have no daemonisation by default or
```
Comment (jrmi) [inline requirements.txt]: Remove comment for now.

File: setup.py [hunk 5]
```diff
@@ -38,6 +38,9 @@
       },
 
       test_suite='nose.collector',
+      install_requires=['begins', 'daemonlite'],
+      # or alternatively w/o daemonlite: -> no daemonization by default
```
Comment (jrmi) [inline setup.py]: Same.

File: pypeman/commands.py (hunk 3 above)
Comment (klausfmh) [reply pypeman/commands.py]: removed try except

File: pypeman/commands.py (hunk 2 above)
Comment (klausfmh) [reply pypeman/commands.py]: removed print

File: pypeman/commands.py (hunk 1 above)
Comment (klausfmh) [reply pypeman/commands.py]: moved lambda to func param defaults

File: requirements.txt (hunk 4 above)
Comment (klausfmh) [reply requirements.txt]: done

File: setup.py (hunk 5 above)
Comment (klausfmh) [reply setup.py]: done

**Pull request merged**